public/js/redux-config.js

```diff
@@ -1,12 +1,9 @@
-'use strict';
-
-var redux = require('redux');
-var dataStore = require('stores/index');
-console.log(dataStore);
+import { createRedux as createRedux_ } from 'redux';
+import * as stores from 'stores';
 
 function createRedux() {
```

Comment:
you could just change this to `export function create()` to avoid the underscore an extra alias.